README.md

```diff
@@ -486,15 +490,13 @@ $ lerna run --scope toolbar-* test
 
 #### --ignore [glob]
 
-Excludes a subset of packages when running the `bootstrap` command.
+Excludes a subset of packages when running a command.
 
 ```sh
 $ lerna bootstrap --ignore component-*
 ```
 
-The `ignore` flag, when used with the `bootstrap` command, can also be set in `lerna.json` under the `bootstrapConfig` key. The command-line flag will take precendence over this option. This flag is supported in `bootstrap` and `exec` commands.
-
-**Note**: If both `scope` and `ignore` are provided to `exec` command, `scope` takes precedence.
```

Comment:
@joscha It changed in this PR, hopefully. Looking through that convo, I don't see a reason you cant accept both flags (as in this PR) and throw an error if the result would return no packages. 

Useful for cases like

```sh
npm install --ignore-scripts && lerna bootstrap --scope "ak*" --ignore "ak-editor*"
```